### DISCUSSION TITLE
Custom PCB for EZ PH, ORP, temp + dosing pump control

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#717] [REM]
I ordered the Whitebox Tentacle and a relay HAT for the PI.  But, I'm thinking I'll do a pcb that will combine those two into a single HAT.  

The Whitebox is "open" and the schematic is readily available.  So, it's trivial to make small modifications and order pcbs for DIY.

Couple of things I'd do different, but I'm interested in comments.

1) I would not connectors for the EZ circuits, but rather solder them (better reliability) to the board.   
2)  Rather than use BNC (or SMA) connectors, I'd use screw type terminal blocks (lower cost, less hassle when replacing probes, and frees up board space for the FETs to control dosing pumps.
3)  Use FETS for controlling the pumps rather than relays (reliability, size, cost).

I plan to include Gerber for this on my next pcb order.  I batch orders to save shipping, since it's usually only $5 for 10 pcbs, but $40 for shipping.   So, it might be a while before I have a couple more projects to include.

Comments?

**Comment by davthomaspilot** [comment]
Here's what I'd use for the dosing pumps (1 chip for 2 pumps):

https://www.mouser.com/ProductDetail/Toshiba/TLP3555AF?qs=%252BEew9%252B0nqrBXhSXGEfjrBQ%3D%3D

**Reply by rstrouse** [reply]
The primary reason for SMA/BNC is that the pH/ORP signals are highly amplified and those connectors are shielded.  Noise in the sensor circuit will make the sampling unreliable so consider where you might have a connector and any place that might generate signal noise.

I like the idea of of a board with a couple of FETs on it.

**Reply by davthomaspilot** [reply]
I see, 

so, the probe cables are shielded with two conductors inside the shield?

The Whitebox schematic seems to indicate two only wires being connected, without a separate, non-current carrying ground being used as a shield. 

![image](https://user-images.githubusercontent.com/6830886/231799560-6686ea4a-426b-4046-8def-27abf222834d.png)

Maybe there's not a separate shield conductor--the return current connector (gnd) is used for the shield?

**Comment by davthomaspilot** [comment]
Here's how it's done in Intellichem:

![20230413_112033](https://user-images.githubusercontent.com/6830886/231808042-a073a6db-5e79-4fd6-bbfb-68784997d2f7.jpg)

**Reply by rstrouse** [reply]
Yeah Atlas also sells an adapter.  I'll be the real signal dangers are floating around on the pad or when the circuit is not isolated.  I have seen a situation where the adapter was the source of unreliable readings.
